cellmlmanip/units.py

```diff
@@ -411,8 +411,6 @@ def _check_unit_of_quantities_equal(self, list_of_quantities):
         :param list_of_quantities: a list of ``pint.Quantity`` objects.
         :returns: boolean indicating whether all units are equivalent
         """
-        if len(list_of_quantities) == 1:
```

Comment:
What happens if you call this with a list of length 0? Think we should just return True if len() < 2. Easy to write a test for!

Reply:
I actually tries a little test:
```
>>> list_of_quantities = []
>>> list_of_quantities = iter(list_of_quantities)
>>> first = next(list_of_quantities, True)
>>> first
True
>>> return all(_is_equal(first, rest) for rest in list_of_quantities)
>>> all(_is_equal(first, rest) for rest in list_of_quantities)
True


>>> from cellmlmanip.model import Quantity
>>> from cellmlmanip.units import UnitStore
>>> list_of_quantities = [Quantity(2, UnitStore().get_unit('dimensionless'))]
>>> list_of_quantities = iter(list_of_quantities)
>>> first = next(list_of_quantities, True)
>>> first
_2
>>> all(_is_equal(first, rest) for rest in list_of_quantities)
True
```

Reply:
Ah! I didn't see the next(.., True) part!